lib/features/popup-menu/ReplaceMenuProvider.js

```diff
@@ -427,18 +427,29 @@ ReplaceMenuProvider.prototype._getLoopEntries = function(element) {
   var translate = this._translate;
 
   function toggleLoopEntry(event, entry) {
-    var loopCharacteristics = getBusinessObject(element).loopCharacteristics;
+    const {
```

Comment:
We should use ES5 (`var`) here.

Reply:
Also, we should not use destructuring.

You can verify things work locally (ES5-style) via 

```
TEST_BROWSERS=PhantomJS npm test
```